leeger/league_loader/YahooLeagueLoader.py

```diff
@@ -59,11 +59,13 @@ def loadLeague(self) -> League:
             raise TimeoutError("Login to yahoofantasy timed out.")
         ctx = Context()
         yahooLeagues = list()
+        curYearLeagueId = self._leagueId
```

Comment:
can we use [snake case](https://khalilstemmler.com/blogs/camel-case-snake-case-pascal-case/#Snake-case) for variables in this PR for consistency?

Reply:
I can, it just seemed like variables introduced locally were using camel case, and only the built in ones from the yahoo api objects were snake lol.

Reply:
I apologize, I got my projects mixed up.

This library does indeed use camel case, you did this correctly.